transport_nantes/topicblog/urls.py

```diff
@@ -10,137 +10,188 @@
 """
 
 app_name = 'topic_blog'
-urlpatterns = [
```

Comment:
Why is commit a488753e part of this PR?  It appears completely unrelated?

It's important not to put irrelevant stuff in PRs.  It just slows down review.  This is something you wrote over a week ago, it was a no brainer approve, and it keeps riding on other PRs, which just delays it getting approved.

Reply:
https://github.com/transport-nantes/tn_web/pull/1053